app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/spark/Ingest.scala

```diff
@@ -323,6 +326,13 @@ object Ingest extends SparkJob with LazyLogging with Config {
     implicit def asS3Payload(status: IngestStatus): String = S3IngestStatus(sceneId, status).asJson.noSpaces
 
     try {
+      Integer.valueOf(Properties.envOrElse("AWS_BATCH_JOB_ATTEMPT", "0")) match {
+        case 3 => {
+          logger.error("Failing early due to previous failures. See prior attempts' logs")
```

Comment:
Is this an area where we'd want to use `sendError`? I could see there being little value in doing so if the idea is that this process is being invoked by the CLI, where this error would bubble up to (and sent to Rollbar).

Reply:
:+1:

Reply:
I used `sendError` in the `catch` block. I queried rollbar for ingest errors, and, if my query is reasonable for finding them, we don't get any currently:

```sql
select * from item_occurrence
where
  (item_occurrence.title like '%Ingest%' or item_occurrence.title like '%ingest%')
  and timestamp > unix_timestamp() - (86400 * 14)
order by timestamp desc;
```

That doesn't seem right, so I made sure that "ingest" appears in the exception that will end up in rollbar.